src/fightPage/buttonsEventListeners.ts

```diff
@@ -0,0 +1,98 @@
+
+    const attackButtonOne = document.querySelector("#attackButtonOne") as HTMLDivElement;
+    const attackButtonTwo = document.querySelector("#attackButtonTwo") as HTMLDivElement;
+    const battleButtons = document.getElementsByClassName("battleButton")! as HTMLCollectionOf<HTMLElement>;
+    const backButton = document.querySelector("#backButton")!;
+
+    attackButtonOne.addEventListener('click', () => {
+      console.log('attack One used!');
+      setTimeout( ()=> {
+        gameHandler.generateActionButtons(player);
+      }, 500);
+      //also change player function 
+      for(let i = 0; i < battleButtons.length; i++) {
+        battleButtons[i].style.backgroundColor = 'green';
+      }
+    });
+    attackButtonTwo.addEventListener('click', () => {
+      console.log('attack Two used!');
+      setTimeout( ()=> {
+        gameHandler.generateActionButtons(player);
+      }, 500);
+      // also change player function
+      for(let i = 0; i < battleButtons.length; i++) {
+        battleButtons[i].style.backgroundColor = 'green';
+      }
+    });
+    backButton.addEventListener('click', () => {
+      //timeout and style for some button animation - the animation needs to be change
+      setTimeout( ()=> {
+        gameHandler.generateActionButtons(player);
+      }, 500);
+      for(let i = 0; i < battleButtons.length; i++) {
+        battleButtons[i].style.backgroundColor = 'green';
+      }
+    });
+  }
+
+  export const switchButtonEventListener = (player: Player, gameHandler: GameHandler) => {
+
+    const switchButtonOne = document.querySelector("#switchButtonOne") as HTMLDivElement;
+    const switchButtonTwo = document.querySelector("#switchButtonTwo") as HTMLDivElement;
+    const battleButtons = document.getElementsByClassName("battleButton")! as HTMLCollectionOf<HTMLElement>;
+    const backButton = document.querySelector("#backButton")!;
+
+    switchButtonOne.addEventListener('click', () => {
+      console.log('pokemon One gooo!');
+      setTimeout( ()=> {
+        gameHandler.generateActionButtons(player);
+      }, 500);
+      // also change player function
+      for(let i = 0; i < battleButtons.length; i++) {
+        battleButtons[i].style.backgroundColor = 'green';
+      }
+    });
+    switchButtonTwo.addEventListener('click', () => {
+      console.log('pokemon Two gooo!');
+      setTimeout( ()=> {
+        gameHandler.generateActionButtons(player);
+      }, 500);
+      // also change player function
+      for(let i = 0; i < battleButtons.length; i++) {
+        battleButtons[i].style.backgroundColor = 'green';
+      }
+    });
+    backButton.addEventListener('click', () => {
+      //timeout and style for some button animation - the animation needs to be change
+      setTimeout( ()=> {
+        gameHandler.generateActionButtons(player);
+      }, 500);
+      for(let i = 0; i < battleButtons.length; i++) {
+        battleButtons[i].style.backgroundColor = 'green';
+      }
+    });
```

Comment:
same here. It might be possible to extract it to one function. As input it could take: 1) button on which the event listener should be added; 2) the function it should call (i.e. eatMango(), switchToPokemon(1) or something)